src/main/java/com/volasoftware/tinder/controller/FriendController.java

```diff
@@ -59,4 +61,15 @@ public ResponseEntity rateFriends(@RequestBody RatingDto ratingDto) {
 
         return ResponseEntity.ok(HttpStatus.OK);
     }
+
+    @GetMapping("/rating")
+    @SecurityRequirement(name = "Bearer Authentication")
+    public ResponseEntity<Page<FriendRatingDetails>> getFriendsSortedByRating(
+            @RequestParam(defaultValue = "0") int page,
```

Comment:
Its a good idea that you added defaultValue to the request params, but also add `required = false`, just to make it available to call the endpoint without request params. Because now I won't be able to call it when I do not provide a request param.